.golangci.yml

```diff
@@ -204,10 +204,14 @@ linters:
       max-func-lines: 0
 
     revive:
+      # Only listed rules are applied
+      # https://github.com/mgechev/revive/blob/HEAD/RULES_DESCRIPTIONS.md
       rules:
+        - name: increment-decrement
```

Comment:
nit: this should be a part of https://github.com/moby/moby/pull/50262/commits/11b057374708323b5cc9f8c8970055202e35c2fa